src/screens/Reports/Report.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { FinancialSummaryRecord } from '../../lib/airtable/interface';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import OutlinedCardList from '../../components/OutlinedCardList';
+import { Typography, List, ListItem, ListItemText } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { RouteComponentProps } from 'react-router-dom';
+import { selectCurrentSiteInformation, round } from '../../lib/redux/siteData';
+import { Person as PersonIcon, ShoppingCart as ShoppingCartIcon, Build as BuildIcon, Warning as WarningIcon } from '@material-ui/icons';
+import ListItemWrapper from '../../components/ListItemWrapper';
+import { useSelector } from 'react-redux';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    section: {
+      padding: '10px 5px',
+    },
+    header: {
+      display: 'inline-flex',
+      alignItems: 'center',
+    },
+    list: {
+      borderLeft: `5px solid ${theme.palette.primary.main}`,
+      padding: '2px 8px',
+      margin: '4px',
+    },
+    icon: {
+      marginRight: '5px',
+    },
+  });
+
+interface ReportProps extends RouteComponentProps {
+  classes: { section: string; header: string; list: string; icon: string; };
+  location: any;
+}
+
+interface Info {
+  label: string;
+  value: string;
+}
+
+function Report(props: ReportProps) {
+  const { classes, match } = props;
+  const report = props.location.state.report;
+  const deadline = props.location.state.deadline;
+  const currentSite = useSelector(selectCurrentSiteInformation);
+
+  // Customers
+  const collected = `${round(report.totalAmountCollected)} Ks`;
+  const billed = `${round(report.totalAmountBilled)} Ks`;
+  const outstanding = `${round(report.totalAmountBilled - report.totalAmountCollected)} Ks`;
```

Comment:
Wait actually I don't think this is how `outstanding` is calculated. On the Figma, this number is bigger than `Billed`. I'll ping ashley on slack and cc you

Reply:
It's bigger and negative, though I thought that was just an error with how data was added